l10n_th_account_tax_filing/models/account_tax_filing.py

```diff
@@ -0,0 +1,405 @@
+# Copyright 2025 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountTaxFiling(models.Model):
+    _name = "account.tax.filing"
```

Comment:
```suggestion
    _name = "account.tax.filing"
    _inherit = ["mail.thread", "mail.activity.mixin"]
```

Add log in new document